src/abstractdataframe/abstractdataframe.jl

```diff
@@ -106,18 +106,18 @@ Set column names
 names!(df::AbstractDataFrame, vals)
 ```
 
-### Arguments
+**Arguments**
```

Comment:
Yeah, it's an unfortunate side-effect of using mkdocs. Not sure way the nicest way to handle this will be until we've got Lapidary generating it's own html/css.